xbmc/Application.h

```diff
@@ -489,24 +477,29 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
   */
   void HandleShutdownMessage();
 
-  CInertialScrollingHandler *m_pInertialScrollingHandler;
+  CInertialScrollingHandler* m_pInertialScrollingHandler;
 
   ReplayGainSettings m_replayGainSettings;
-  std::vector<IActionListener *> m_actionListeners;
-  std::vector<std::string> m_incompatibleAddons;  /*!< Result of addon migration */
+  std::vector<IActionListener*> m_actionListeners;
+  std::vector<std::string> m_incompatibleAddons; /*!< Result of addon migration */
   std::deque<XBMC_Event> m_winEvents;
 
 private:
-  CCriticalSection m_critSection;                 /*!< critical section for all changes to this class, except for changes to triggers */
-
-  CCriticalSection m_frameMoveGuard;              /*!< critical section for synchronizing GUI actions from inside and outside (python) */
-  std::atomic_uint m_WaitingExternalCalls;        /*!< counts threads wich are waiting to be processed in FrameMove */
-  unsigned int m_ProcessedExternalCalls;          /*!< counts calls wich are processed during one "door open" cycle in FrameMove */
-  unsigned int m_ProcessedExternalDecay = 0;      /*!< counts to close door after a few frames of no python activity */
+  CCriticalSection
+      m_critSection; /*!< critical section for all changes to this class, except for changes to triggers */
```

Comment:
what weird style is this? looks ugly imo

Reply:
Yes, this is an edge case. It's set to never break comments because it does it badly so it tries to keep the line length by wrapping the code instead. I think the solution here is to manually wrap the comment to avoid this weird style